packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -310,19 +310,21 @@ export class AtomicFacet
     }
 
     if (!this.facetState.values.length) {
-      return;
+      return <Host class="without-values"></Host>;
```

Comment:
In the red diff, when there were no values, the component would return `undefined`.

I'm trying to understand why the component was taking up space in this case?

Reply:
The contents would be empty, but the component is still there. I'm hiding it OOTB **but** the main purpose would be to use the class to check "do I have any visible facets in my interface?" If yes, then a customer could adapt their UI to hide a section, for example make the result list take the entire page.

Reply:
I'm still not clear on where the width is coming from. Is it that the facet components have a default width? Or is it coming from the facet manager?

Adding a class is a possible solution to the problem, but as you said, it introduces the possiblity there will be CSS conflicts. It also requires the client to add certain rules versus having the sections adapt OOTB.

Reply:
I thought OOTB meant 'out of the box' but after reading your comments, I am not so sure.

Reply:
I'll prepend Atomic.

> where the width is coming from

It's coming from the content of the facet. If you don't set a width, the facet column will resize depending on the facet content. With this feature, you could set a static with on all facet *if* they have values. I'll prepend atomic- to the class so it remove unlikely but potential issues.